src/session.rs

```diff
@@ -91,23 +91,25 @@ pub enum MediaEvent {
 }
 
 impl Session {
-    pub fn new(codec_config: CodecConfig, ice_lite: bool, use_bwe: bool) -> Self {
+    pub fn new(
+        codec_config: CodecConfig,
+        ice_lite: bool,
+        use_bwe: bool,
+        bwe_initial_estimate: Bitrate,
+    ) -> Self {
         let mut id = SessionId::new();
         // Max 2^62 - 1: https://bugzilla.mozilla.org/show_bug.cgi?id=861895
         const MAX_ID: u64 = 2_u64.pow(62) - 1;
         while *id > MAX_ID {
             id = (*id >> 1).into();
         }
         let (pacer, bwe) = if use_bwe {
-            let initial_bitrate = 300_000.into();
-            let mut pacer = PacerImpl::LeakyBucket(LeakyBucketPacer::new(
-                initial_bitrate * PACING_FACTOR * 2.0,
+            let pacer = PacerImpl::LeakyBucket(LeakyBucketPacer::new(
+                bwe_initial_estimate * PACING_FACTOR * 2.0,
                 Duration::from_millis(40),
             ));
 
-            pacer.set_padding_rate(initial_bitrate * PADDING_FACTOR);
```

Comment:
We always want padding to be set in the same way, I don't see why we should remove this

Reply:
if you have no desired bitrate to reach, why would you have a padding target ?

Reply:
Initial bitrate == desired bitrate. If we don't pad the estimates will be lies, for example if the user says `initial_bitrate: Bitrate::kbps(1500)` but then sends only 750Kbit/s of media the resulting estimates will not be reliable since we aren't actually sending near the estimate

Reply:
Either case might be a bit moot since the user will probably reconfigure the padding and pacing rate immediately anyway.

One thing to consider is what happens if we make BWE ignore audio packets, as we intend to. Then I think it might be suitable to set the desired bitrate to something like `expected_ingress_video_rate * 1.15`, thus instructing the BWE system to discover if we have enough bitrate for even a single video track. 

With all of this maybe we should just add an explicit desired_bitrate argument to this method as well?

Reply:
Not sure I understand to which method we would add desired_bitrate and why.

I do understand having a margin for audio, which can be taken into account when calling set_bwe_desired_bitrate()

Reply:
ah, i see the other comment

Reply:
I'm suggesting this method should accept both desired and initial rates. The idea is to ignore audio entirely for BWE purposes.

However if we start with only audio I think we'd want the initial values to be something like 

```
config.enable_bwe(Bitrate::kbps(1500), Bitrate::kbps(1800)
```

As you said before we start at 1.5Mbit/s because that's faster than ramping up from 300Kbit/s and we'll quickly decrease the estimate if we are exceeding. Typically, at least for our use case, we'll initially only be sending an audio track but we still want to discover enough bitrate to later receive some video tracks so we'd set the desired to something that could sustain either a high quality video track or a few lower quality tracks. This way, when we do receive some video tracks we'll have already discovered some bitrate

Reply:
lets continue this in the other "thread" below